crates/re_viewer/src/store_hub.rs

```diff
@@ -275,6 +275,28 @@ impl StoreHub {
         self.store_bundle.contains_recording(id)
     }
 
+    /// Remove any recordings with a network source pointing at this `uri`.
+    #[cfg(target_arch = "wasm32")]
+    pub fn remove_recording_by_uri(&mut self, uri: &str) {
+        self.store_bundle.entity_dbs.retain(|_, db| {
+            let Some(data_source) = &db.data_source else {
+                // no data source, keep
+                return true;
```

Comment:
Does this make sense? I'm not sure how it's possible to have a database without a data source, and if it should be handled differently here.

Reply:
Yeah -- I'd love for us to refactor things to make this not possible. Mostly this shows up in unit-tests with programmatically created stores.

Any data being streamed into the app should have it's store set based on the channel-source though:
See: https://github.com/rerun-io/rerun/blob/main/crates/re_viewer/src/app.rs#L819-L821